### PR TITLE
Sphinx entry

### PR DESCRIPTION
**Pull request by jo-mueller**
Hi all,

I wrote a documentation article on how to use Sphinx to auto-generate documentation pages with Sphinx. I'd be happy if you could crossread it at some time :)

### CODE REVIEW

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:4]: Typos in the word 'proper' and  'the last few meters'

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:46]: typo in the word 'call'

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:146]: typo in the word 'difficult'

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:150]: typo in the word contains here and in line 276

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:34]: missing of after the 2nd consists: (`subpackage_a`) that consists function definitions

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:192]: typo in word 'writing'

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:206]: nnotebooks

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:266]: mmathematical

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:295]: seemless -> seamless

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:302]: shhould

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:305]: en empty file -> an empty file

**Review by lazigu — CHANGES_REQUESTED** [review]
Sorry, I ended up marking quite a few typos :D I guess it would have been faster to fix them as I go myself, but I am not familiar with blog post reviews and what is the correct way to mark typos.  Let me know if I should do it differently for the next time!

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:86]: modules **are** classes -> and classes?

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:95]: ccurrently -> currently

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:119]: `These *.rst* files (so-called "stubs") **to** are used to tell Sphinx what should be documented. ` I think this 'to' is unnecessary.

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:161]: typo where those three commas ,,, are

File: johannes_mueller/entry_sphinx/Readme.md
Comment (lazigu) [inline johannes_mueller/entry_sphinx/Readme.md:261]: includes

**Comment by lazigu** [comment]
A couple of more comments because I found out that I can view it as a rich text and noticed a bit more :)
Sorry for the comments spam!

**Comment by jo-mueller** [comment]
Thanks for looking up the typos ^^ I feel a little dyslexic right now!

I think there is a way to mark changes in the text straight away so I can just commit suggest changes, but I'm not exactly sure how.

File: johannes_mueller/entry_sphinx/Readme.md
Comment (zoccoler) [inline johannes_mueller/entry_sphinx/Readme.md:6]: Loved Patrick's images 🤣 
Can we use them?

File: johannes_mueller/entry_sphinx/Readme.md
Comment (zoccoler) [inline johannes_mueller/entry_sphinx/Readme.md]: I suggest putting this first phrase as the first phrase of the next topic.

Comment (zoccoler) [reply johannes_mueller/entry_sphinx/Readme.md]: Also, maybe 
`conda install -c anaconda sphinx`
so we stick to conda as much as possible inside a conda environment?

Comment (zoccoler) [reply johannes_mueller/entry_sphinx/Readme.md]: obs: docs directory already exists when you clone the repository

File: johannes_mueller/entry_sphinx/Readme.md
Comment (zoccoler) [inline johannes_mueller/entry_sphinx/Readme.md:138]: where is step 4.?

File: johannes_mueller/entry_sphinx/Readme.md
Comment (zoccoler) [inline johannes_mueller/entry_sphinx/Readme.md]: Did you mean 'sometimes' or really 'sometime'

File: johannes_mueller/entry_sphinx/Readme.md
Comment (zoccoler) [inline johannes_mueller/entry_sphinx/Readme.md:292]: This is awesome!! 🚀

**Review by zoccoler — APPROVED** [review]
Sphinx looks amazing! 
My comments are just minor stuff.
Thanks for digging into that and explaing it here, it worked like charm!

**Review by lazigu — APPROVED** [review]

File: johannes_mueller/entry_sphinx/Readme.md
Comment (jo-mueller) [reply johannes_mueller/entry_sphinx/Readme.md]: I removed it from the example directory

File: johannes_mueller/entry_sphinx/Readme.md
Comment (jo-mueller) [reply johannes_mueller/entry_sphinx/Readme.md:223]: Removed it from there for clarity :)

**Comment by jo-mueller** [comment]
Thanks for your feedback! I have integrated all suggested changes, can I merge it?
[x] Laura
[ ] Marcelo
[ ] Robert

File: johannes_mueller/entry_sphinx/Readme.md
Comment (jo-mueller) [reply johannes_mueller/entry_sphinx/Readme.md:6]: Hm...I would assume that typical memes are under CC license as they are naturally shared criss-cross the internet. The template is, for instance, available at meme-generator (https://imgflip.com/meme/183221332/PAtrick-Smart-Dumb), which doesn't bother with licenses at all ^^"

**Comment by jo-mueller** [comment]
@zoccoler @haesleinhuepf what do you think?
I would like to have 2/3 approvals before I merge  :)

File: johannes_mueller/entry_sphinx/Readme.md
Comment (zoccoler) [reply johannes_mueller/entry_sphinx/Readme.md:6]: Ok great! :)

**Comment by zoccoler** [comment]
All good for me! 👍

File: johannes_mueller/entry_sphinx/Readme.md
Comment (zoccoler) [reply johannes_mueller/entry_sphinx/Readme.md]: I managed to do it now! I have no idea why I couldn't earlier... I guess seeing the image helped, thx :)

**Comment by haesleinhuepf** [comment]
Hi Johannes @jo-mueller ,

great work. Thanks for writing this up! 

Also thanks to Laura and Marcelo for reviewing. Great team work 😃


Best,
Robert

**Comment by haesleinhuepf** [comment]
One more comment Johanned @jo-mueller , I had to remove one pictrue where you didn't own the copyright.

**Pull request merged**
